pkg/controller/servicebindingrequest/reconciler.go

```diff
@@ -129,13 +137,16 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 
 	selectors := extractServiceSelectors(sbr)
 	if len(selectors) == 0 {
-		conditionsv1.SetStatusCondition(&sbr.Status.Conditions, conditionsv1.Condition{
-			Type:    BindingReady,
-			Status:  corev1.ConditionFalse,
-			Reason:  EmptyServiceSelectorsReason,
-			Message: "The spec.backingServiceSelectors field is empty.",
-		})
-		_, updateErr := updateServiceBindingRequestStatus(r.dynClient, sbr)
+		_, updateErr := updateServiceBindingRequestStatus(
+			r.dynClient,
+			sbr,
+			conditionsv1.Condition{
+				Type:    InjectionReady,
```

Comment:
Should we have collection Ready and injection ready both as false in case of len(selectors) ==0 because ideally when we don't have any service then no collection is happening.

Reply:
+1 I will add it.

Reply:
Is this okay @Avni-Sharma 9b395a8 ?

Reply:
looks good :+1: